Repository cleanup, sdk update and gitignore

mrvux opened the pull request:
As mentioned in https://github.com/Microsoft/prepose/issues/2

I added the first two improvements and also cleaned up build folders (I tested a clean rebuild and sample is running as expected with latest sdk version)

Also I added a try/catch block in the sample, and display a generic error message if compilation failed (to avoid crash)

Many thanks
Julien

mrvux committed:
[Cleanup] Remove all bin/obj folders from

mrvux committed:
[Build] Update Kinect dependency to latest sdk version

[Misc] Add gitignore to avoid committing intermediate files

mrvux committed:
Merge branch 'master' of github.com:Microsoft/prepose

mrvux committed:
[Sample] Add a compile status textblock

[Sample] Embed script compilation in a try/catch block, and display error message if compilation failed.

too4words commented:
We will clean up the error messages when I get a chance. Thanks for your help!

too4words commented:
I am also curious about how you would like to/plan to use Prepose. We are interested to see if our trajectories intersect.

mrvux commented:
First thing is to integrate it into various creative coding toolsets and prototyping environments.

I'm already maintaining dx11 rendering as well as Kinect2 within vvvv (vvvv.org) , so artists can use it for various purposes, so I added support for prepose there, which will be available in the coming week (there will be some more pull requests as I fixed the reader code for that purpose).
